voxblox_ros/src/transformer.cc

```diff
@@ -143,29 +147,50 @@ bool Transformer::lookupTransformQueue(const ros::Time& timestamp,
     }
   }
 
+  // Match found basically means an exact match.
```

Comment:
Should we reduce the timestamp_tolerance now that we have interpolation, make it like 1ms?